### PR TITLE
Programming Languages - Python Project section

### PR DESCRIPTION
**Pull request by Varsha010101**
Programming Languages - Python Project section

### Description: 
This pull request adds python projects to build up a resume. It has projects at beginner, intermediate and advanced level. It provides all resources including source code, documentation and tutorials 

-https://github.com/topics/python-blockchain
-https://pythongeeks.org/python-library-management-system-project/
-https://www.geeksforgeeks.org/real-time-translation-app-using-python/
-https://www.geeksforgeeks.org/gui-chat-application-using-tkinter-in-python/
-https://realpython.com/build-a-chatbot-python-chatterbot/
-https://www.geeksforgeeks.org/data-visualization-with-python/
-https://pythongeeks.org/python-music-player/
-https://cointrader.readthedocs.io/en/latest/_modules/cointrader/chart.html

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Varsha010101** [comment]
@jfmartinz Kindly label this PR under GSSoC.

File: Programming Languages/Python/readme.md [hunk 1]
```diff
@@ -566,4 +570,180 @@ Automate repetitive tasks with Python using these useful resources.
...
+> Python Projects which can help you to enhance your development skills and build your resumes.
+
+### Beginner Level Projects
+
+
+<table>
+  <tr>
+    <th>Project Name</th>
+    <th>Description</th>
+    <th>Resources</th>
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:585]: The order of the col should be:
1. Resource Name
2. Description

Remove the Project Name and add the URL into the Resource Name

for example:

```html
<td><a href="URL">Resource Name </a> </td>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by Varsha010101** [comment]
Yes, I have made the requested changes kindly review it. @jfmartinz

**Review by jfmartinz — CHANGES_REQUESTED** [review]
The heading is too small:

image

File: Programming Languages/Python/readme.md [hunk 2]
```diff
@@ -567,39 +573,180 @@ Automate repetitive tasks with Python using these useful resources.
         <td>Mosh Hamedani delivers Python tutorials focusing on practical programming skills, software development best practices, and tips for becoming a proficient Python developer.</td>
       </tr>
 </table>
+These channels provide excellent resources for learning Python programming, catering to learners of all levels with their diverse range of content and teaching styles.
 
-### YouTube Playlist
+#### Python Projects
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:578]:
```suggestion
### Python Projects
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by jfmartinz** [comment]
Still small. Can you check at your end?

File: Programming Languages/Python/readme.md [hunk 3]
```diff
@@ -576,9 +576,10 @@ Automate repetitive tasks with Python using these useful resources.
       </tr>
 </table>
 These channels provide excellent resources for learning Python programming, catering to learners of all levels with their diverse range of content and teaching styles.
-<head>
 
-#### Python Projects
+
+# Python Projects
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md]:
```suggestion
### Python Projects
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Review by jfmartinz — APPROVED** [review]
Thanks @Varsha010101

**Comment by jfmartinz** [comment]
Can you please solve the conflict

File: Programming Languages/Python/readme.md [hunk 4]
```diff
@@ -129,8 +133,9 @@ Object-Oriented Programming (OOP) concepts in Python with these tutorials coveri
 ### Core Python Libraries
 Core Python Libraries with these tutorials covering strings, collections framework, exception handling, file handling, input/output (I/O), and multithreading.
 
+
 <table width="100%">
-  <tr>
+  <tr>https://github.com/jfmartinz/ResourceHub/pull/446/conflict?name=Programming%2BLanguages%252FPython%252Freadme.md&ancestor_oid=ce12fab1fd551f6f14b3dd9b47e739d674220cb2&base_oid=a312ecd67ff407e8d084c8feea59a74572d46ff3&head_oid=a82d3d60d00f7ef27df58f05c5d86d43bfaeda98
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:138]:
```suggestion
  <tr>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Programming Languages/Python/readme.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+<Head>
+
+
+
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:4]:
```suggestion
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Check Java file:
image

Please remove it.

In Python, you remove:

image

In Youtube Playlist

File: Programming Languages/Python/readme.md [hunk 6]
```diff
@@ -748,6 +746,7 @@ These channels provide excellent resources for learning Python programming, cate
 
   
 =======
+
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:749]:
```suggestion
### YouTube Playlists
```

File: Programming Languages/Python/readme.md [hunk 7]
```diff
@@ -748,6 +746,7 @@ These channels provide excellent resources for learning Python programming, cate
 
   
 =======
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md]:
```suggestion

```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by Varsha010101** [comment]
The changes mentioned are already done. Could you please tell if any other changes required.

**Comment by jfmartinz** [comment]
Can please raise another PR, the PR is already messy and there is a conflict.

**Pull request closed without merging**